### DISCUSSION TITLE
我這邊的網路故障了

### DISCUSSION THREAD

**Discussion by bestian** [3dw/auto20-next#139] [General]
Hi, @leechiuhui 
我這邊的室內網路故障了, 手機網路沒有吃到飽
今天修好兩個issues後, 在網路修好前暫時無法測試和build, 先說一聲~

**Comment by leechiuhui** [comment]
Hi,teacher.Today is a scheduled workday for 3DW, which typically involves working on Tuesdays and Fridays. However, since I have to catch a flight today, I will only be focusing on studying the code on the traveling.

**Reply by bestian** [reply]
It's ok, 不急.
我這邊的網路也好了.
